pkg/reconciler.v1/common/interface.go

```diff
@@ -239,21 +239,3 @@ type JobInterface interface {
 	// PastActiveDeadline CAN be overridden to customize how to determine if this job has past activate deadline.
 	PastActiveDeadline(runPolicy *commonv1.RunPolicy, jobStatus *commonv1.JobStatus) bool
 }
-
-// KubeflowReconcilerInterface defines the abstract interface for a base reconciler for kubeflow jobs.
-type KubeflowReconcilerInterface interface {
```

Comment:
What's the motivation to move `KubeflowReconcilerInterface` to `training-operator`?

Reply:
When writing the test code, `test_reconciler`, which could be considered as one example of how the `common.reconciler` will be used, I found that:

1. Such interface will not be used for any abstraction when implementing a real reconciler
2. Embedding modularized interfaces in `KubeflowReconcilerInterface` conflicts embedding them in the real reconciler struct.

Reply:
I think it make sense to me. current testing integration between common and training-operator is a headache.